src/scripts/ui/menu/index.ts

```diff
@@ -165,15 +166,36 @@ export class ComfyAppMenu {
       experimental: true,
       tooltip: 'On small screens the menu will always be at the top.',
       type: 'combo',
-      options: ['Disabled', 'Top', 'Bottom'],
+      options: ['Disabled', 'Floating', 'Top', 'Bottom'],
       onChange: async (v: MenuPosition) => {
```

Comment:
Can the whole menu position logic get moved to Vue side?

Reply:
I don't think this is worthwhile now as I don't think we're planning on supporting 4 different menus long term?